bin/mail/2016-04-26-changeorg-petition/generate-lists.rb

```diff
@@ -0,0 +1,22 @@
+#!/usr/bin/env ruby
+require_relative '../mailing-common/mailing-list-utils'
+
+# All US contacts
+us_contacts_query = %q(
+  !(role_s: student) && (
+    create_ip_country_s: "United States" ||
+    hoc_country_s: "us" ||
+    location_country_code_s: "US" ||
+    location_country_s: "United States"
+  )
+)
```

Comment:
Yes. Many of them are duplicates. After filtering out unsubscribers and deduping, which is done [here](https://github.com/code-dot-org/code-dot-org/blob/staging/bin/mail/mailing-common/mailing-list-utils.rb#L75-L75), the final count is 830590.

This seemed the simplest way to get all US contacts. I'm definitely open to other suggestions.